docs/advanced-application/quiz-toolkit.md

```diff
@@ -0,0 +1,189 @@
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "your-client-x509-cert-url"
+}
+```
+
+The contents of `credentials.json` will be used as the `service_account_info` parameter. Simply copy this information and pass it to the `append_to_sheet()` function as the `service_account_info` argument.
+
+
+## Steps 3: Send email to someone when they submit.
+
+You can use `send_gmail` function.
+#### Arguments:
+
+| Argument            | Type     | Description                                                                |
+|---------------------|----------|----------------------------------------------------------------------------|
+| **`sender_email`**      | **str**  | The email address of the sender.                                           |
+| **`receiver_email`**    | **str**  | The email address of the receiver.                                         |
+| **`subject`**           | **str**  | The subject line of the email.                                             |
+| **`app_password`**      | **str**  | The app-specific password for the sender's email account.                  |
+| **`values`**            | **list** | A list of lists. Each list contains data to be included in the email body. |
+
+For example, you can retrieve the student's `name` and `id` from the input, calculate the `grade` through code, and this information can be sent to the receiver's email. All you need to do is add :
+```python
+values = [
+        ["John", "123456", 96]
+]
+msc.send_email(sender_email = "sender_email@gmail.com", 
+               receiver_email = "receiver_email@example.ca", 
+               subject = "Test Result", 
+               app_password = "xxxx xxxx xxxx xxxx", 
+               values = values)
+```
+Then receiver will get an email like this:
+
+![img alt](/docs/advanced-application/quiz-toolkit/gmail_ex.png)
+
+In this part, we will teach you how to find the `app_password` parameter for `send_gmail` function
+### How to Create a Google App Password
+
+Official Google support page: [Create & use App Passwords](https://support.google.com/accounts/answer/185833?hl=en).
+
+An app password is a 16-digit code that allows less secure apps or devices to access your Google Account. This is especially useful if you have 2-Step Verification enabled and the app you're using doesn't support "Sign in with Google." Here’s how to create one:
+
+1. Go to your [Google Account](https://myaccount.google.com/).
+2. Select **Security.**
+3. Under "Signing in to Google," select **2-Step Verification**.
+   ![img alt](/docs/advanced-application/quiz-toolkit/gmail_step_1.png)
+4. At the bottom of the page, select **App passwords**.
+5. Enter a name that helps you remember where you’ll use the app password.
+6. Select **Generate**.
+7. To enter the app password, follow the instructions on your screen. The app password is the 16-character code that generates on your device.
```

Comment:
Not sure about this. Maybe something like "Make sure to save the app password (as you won't be able to see it later) and then copy it into your quiz app"